src/mobile/Header/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import styles from './Header.scss';
+import cn from 'classnames';
+
+interface Props {
+  logo: React.ReactElement;
+  searchBar: React.ReactElement;
+  languages: React.ReactElement;
+  burger: React.ReactElement;
+}
+
+export function Header({ logo, searchBar, languages, burger }: Props) {
```

Comment:
1. → HeaderProps
2. You can avoid using props. This props will never change, so you could use needed components in `Header` component.
3. Better to name it MobileHeader and rename Header for desktop as DesktopHeader